_episodes/03-multiple_files.md

```diff
@@ -73,15 +73,14 @@ print(filenames)
 This will give us a list of all the files which end in `*.out` in the `outfiles` directory. Now if we want to parse every file we just read in, we will use a `for` loop to go through each file.
 ```
 for f in filenames:
-    outfile = open(f,'r')
-    data = outfile.readlines()
-    outfile.close()
-    for line in data:
-        if 'Final Energy' in line:
-            energy_line = line
-            words = energy_line.split()
-            energy = float(words[3])
-            print(energy)
+    with open(f,'r') as data:
```

Comment:
I think the idea of switching to the `with` statement is a good idea. @armcdona and I will have to discuss this.

Reply:
I have open issue #60 for this!